lib/pages/new_group/new_group_chat_info.dart

```diff
@@ -47,14 +52,16 @@ class NewGroupChatInfoController extends State<NewGroupChatInfo>
   final haveGroupNameNotifier = ValueNotifier(false);
   final createRoomStateNotifier =
       ValueNotifier<Either<Failure, Success>>(Right(CreateNewGroupInitial()));
+  final ValueNotifier<Either<Failure, Success>> getAvatarNotifier =
+      ValueNotifier<Either<Failure, Success>>(Right(NewGroupInitialUIState()));
   final uploadContentInteractor = getIt.get<UploadContentInteractor>();
   final uploadContentWebInteractor =
       getIt.get<UploadContentInBytesInteractor>();
   final createNewGroupChatInteractor =
       getIt.get<CreateNewGroupChatInteractor>();
   final groupNameTextEditingController = TextEditingController();
   final avatarAssetEntityNotifier = ValueNotifier<AssetEntity?>(null);
-  final avatarFilePickerNotifier = ValueNotifier<FilePickerResult?>(null);
+  MatrixFile? avatarFilePickerNotifier;
```

Comment:
IMO, We should take care of releasing it `avatarFilePickerNotifier =  null` when not in use to avoid memory leaks.

Reply:
Updated